src/views/components/LoginForm/__tests__/container.spec.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import LoginForm, { handleSubmit } from '../container'
+
+describe('LoginForm to match snapshot', () => {
+  it('with default props', () => {
+    const container = shallow(<LoginForm />)
+    const props = {
+      onSubmit: jest.fn()
+    }
+
+    expect(
+      handleSubmit({ email: 'tset', password: 'test' }, { props })
+    ).toMatchSnapshot()
+    expect(container).toMatchSnapshot()
```

Comment:
Don't use multiple `expect` in single `it`

Reply:
Why cant we use multiple expect? If I want to check all cases for part functional

Reply:
But in this case useless second expect, I remove it